request_helpers_test.go

```diff
@@ -161,6 +165,8 @@ func TestRequestHelpers_delete(t *testing.T) {
 }
 
 func TestRequestHelpers_paginateAll(t *testing.T) {
+	const totalResults = 4123
```

Comment:
I made this much larger and uneven to reproduce the failure case before applying the fix